lib/inquisition/fasterer/runner.rb

```diff
@@ -0,0 +1,35 @@
+require 'fasterer/file_traverser'
+
+module Inquisition
+  module Fasterer
+    class Runner < ::Inquisition::Runner
+      attr_reader :errors
+
+      def call
+        @errors = []
+        fasterer = ::Fasterer::FileTraverser.new('.')
+        fasterer.scannable_files.each { |file| scan_file(file) }
+        errors
+      end
+
+      private
+
+      def scan_file(path)
+        analyzer = ::Fasterer::Analyzer.new(path)
+        analyzer.scan
+        errors << create_issue(analyzer) if analyzer.errors.any?
+      end
+
+      def create_issue(data)
+        current_error = data.errors.instance_variable_get(:@offenses).first
```

Comment:
Is it possible that there will be more than one offenses?
Also maybe there is some interface instead of using `instance_variable_get `